R/tokens_ngrams.R

```diff
@@ -48,8 +48,7 @@ tokens_ngrams.default <- function(x, n = 2L, skip = 0L, concatenator = "_") {
 ## the grammatical rules of quanteda (inputs character, outputs tokens),
 ## but starts with "tokens_"
 #' @importFrom stats complete.cases
-tokens_ngrams.character <- function(x, n = 2L, skip = 0L, concatenator = "_") {
-
+tokens_ngrams_character <- function(x, n = 2L, skip = 0L, concatenator = "_") {
```

Comment:
Can you remind me why we need `tokens_ngrams_character()`? It if is only for tests, we should move to `test-tokens_ngrams.R`.

Reply:
You're right, it's no longer used anywhere, and it's not even tested. I'll delete it.